pkg/oc/cli/admin/release/extract.go

```diff
@@ -147,3 +206,171 @@ func (o *ExtractOptions) Run() error {
 		return opts.Run()
 	}
 }
+
+func (o *ExtractOptions) extractGit(dir string) error {
+	if err := os.MkdirAll(dir, 0750); err != nil {
+		return err
+	}
+
+	release, err := NewInfoOptions(o.IOStreams).LoadReleaseInfo(o.From)
+	if err != nil {
+		return err
+	}
+
+	cloner := &git{}
+
+	hadErrors := false
+	alreadyExtracted := make(map[string]string)
+	for _, ref := range release.References.Spec.Tags {
+		repo := ref.Annotations["io.openshift.build.source-location"]
```

Comment:
since we rely on these on multiple places, I would put them in `openshift/api` ?

/cc @bparees

Reply:
We don't do that for constants usually.  I can pull them up to a package local private though.

Reply:
And will add comments.